synapse/models/orgs.py

```diff
@@ -223,6 +223,10 @@ def getModelDefs(self):
                     'doc': 'The contribution includes the specific node.'}),
                 ((None, 'meets', 'ou:requirement'), {
                     'doc': 'The requirement is met by the source node.'}),
+                (('ou:org', 'has', None), {
+                    'doc': 'The organization is or was in posession of target node.'}),
+                (('ou:org', 'owns', None), {
```

Comment:
maybe worth adding for the other applicable entities? I'm thinking some or all of ou:suborg, ou:team, ou:position, ou:user?